source/github-helpers/hotkey.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'dom-chef';
+
+export function registerHotkey(hotkey: string, action: VoidFunction | string): Deinit {
+	const element = typeof action === 'string'
+		? <a hidden href={action} data-hotkey={hotkey}/>
```

Comment:
I also added support for strings, but haven't found any use for it where it would simplify the code.

Reply:
What if we treat this string as a selector and do `select(action).click()`?

To emulate link behavior, just pass an action of `() => location.assign(link)`.

Reply:
> What if we treat this string as a selector and do `select(action).click()`?

addHotkey already does that:
```js
registerHotkey('ctrl-a', 'a[blah]') // Your suggestion
addHotkey(select('a[blah]'), 'ctrl-a') // Already-possible solution
```

> To emulate link behavior, just pass an action of `() => location.assign(link)`.

There's no substitute for real links (which can be handled by Turbo as well).

I do see two possible improvements here:

- make the two functions consistent (always accept "hotkey" first), or 
- merge the two functions so `registerHotkey` accepts an element passed as "action"